TrueCraft.Client/TrueCraftGame.cs

```diff
@@ -38,6 +38,7 @@ public class TrueCraftGame : Game
         public DateTime StartDigging { get; set; }
         public DateTime EndDigging { get; set; }
         public Coordinates3D TargetBlock { get; set; }
+        public int HotbarSelection { get; set; }
```

Comment:
This probably belongs in MinecraftClient as a property that informs the server of the change.